src/hooks/use-canvas.ts

```
@@ -759,4 +759,244 @@ export const useCanvas = () => {
+
+export const isShapeInsideFrame = (shape: Shape, frame: FrameShape): boolean => {
+    const frameLeft = frame.x
+    const frameTop = frame.y
+    const frameRight = frame.x + frame.w
+    const frameBottom = frame.y + frame.h
+
+    switch(shape.type) {
+        case 'rect':
+        case 'ellipse':
+        case 'frame':
+            const centerX = shape.x + shape.w / 2
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)